src/distilabel/pipeline/llm/openai.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Optional, Union
+
+from openai import OpenAI
+from pydantic import field_validator
+
+from distilabel.pipeline.llm.base import LLM
+from distilabel.pipeline.step.task.types import ChatType
+
+
+class OpenAILLM(LLM):
+    model: str = "gpt-3.5-turbo"
+    api_key: Optional[str] = os.getenv("OPENAI_API_KEY")
```

Comment:
Question here, do we want `api_key` as pydantic's [`SecretStr`](https://docs.pydantic.dev/latest/examples/secrets/)?